escalation-eda/escalation-swf/README.md

```diff
@@ -66,10 +80,14 @@ The sample implementation using the Atlassian JIRA service is defined by the fol
 similar fields are subject to translation to the configured language and cannot be used for a consistent check.
 
 ### Dependencies on latest SonataFlow artifacts
-The current implementation depends on version `2.0.0-SNAPSHOT` of the SonataFlow platform artifacts as you can see in the [pom.xml](./pom.xml). 
+* Earlier the implementation was dependent on version `2.0.0-SNAPSHOT` of the SonataFlow platform artifacts. However, this version is no longer supported or exists. 
```

Comment:
I think we can get rid of old dependency info

Reply:
Done. Thanks.